docs/index.asciidoc

```diff
@@ -50,10 +51,80 @@ filter plugins.
   * Value type is <<array,array>>
   * There is no default value for this setting.
 
-A new clone will be created with the given type for each type in this list.
+ECS Compatibility disabled: a new clone will be created with a `type` of the given type in this list.
+
+ECS Compatibility enabled: a new clone will be created with a `tag` of the given type in this list.
+
+Example:
+[source,ruby]
+    filter {
+      clone {
+        clones => ["sun", "moon"]
+      }
+    }
+
+ECS disabled
+[source,text]
+-----
+{
+      "@version" => "1",
+      "sequence" => 0,
+       "message" => "Hello World!",
+    "@timestamp" => 2021-03-24T11:20:36.226Z,
+          "type" => "sun",
+          "host" => "example.com"
+}
+{
+      "@version" => "1",
+      "sequence" => 0,
+       "message" => "Hello World!",
+    "@timestamp" => 2021-03-24T11:20:36.226Z,
+          "type" => "moon",
+          "host" => "example.com"
+}
+-----
+
+ECS enabled
+[source,text]
+-----
+{
+          "tags" => [
+        [0] "sun"
+    ],
+      "sequence" => 0,
+    "@timestamp" => 2021-03-23T20:25:10.042Z,
+       "message" => "Hello World!",
+      "@version" => "1",
+          "host" => "example.com"
+}
+{
+          "tags" => [
+        [0] "moon"
+    ],
+      "sequence" => 0,
+    "@timestamp" => 2021-03-23T20:25:10.042Z,
+       "message" => "Hello World!",
+      "@version" => "1",
+          "host" => "example.com"
+}
+-----
 
 Note: setting an empty array will not create any clones. A warning message is logged.
 
+[id="plugins-{type}s-{plugin}-ecs_compatibility"]
+===== `ecs_compatibility`
+
+* Value type is <<string,string>>
+* Supported values are:
+** `disabled`: unstructured geo data added at root level
+** `v1`: uses fields that are compatible with Elastic Common Schema
+* Default value depends on which version of Logstash is running:
+** When Logstash provides a `pipeline.ecs_compatibility` setting, its value is used as the default
+** Otherwise, the default value is `disabled`.
+
+Controls this plugin's compatibility with the {ecs-ref}[Elastic Common Schema (ECS)].
+The value of this setting affects the _default_ value of `type` and `tags`, see <<plugins-{type}s-{plugin}-clones>>
```

Comment:
this sounds weird, I understand your intention being familiar with the code.
maybe @karenzone can help, maybe:

> The value of this setting affects the behavior of the <<plugins-{type}s-{plugin}-clones>> option`

and than mention the `type` vs `tags` behavior there?